### PR TITLE
Add images in >poll and >polltally

### PR DESCRIPTION
**Pull request by keli5**
Just attach an image to the >poll command, and enjoy.

- Could also throw an error if there's more than one image, if you wanted? (but why?)
- Just Works™ if the attachment's not an image

### CODE REVIEW

File: crimsobot/cogs/utilities.py [hunk 1]
```diff
@@ -36,11 +36,16 @@ async def poll(self, ctx: commands.Context, *, poll_input: str) -> None:
 
         choices = poll_input.split(';')
         question = choices.pop(0).strip()
+        attachment = None
 
         # some tests to ensure there's a question and at least two choices
         if len(question) == 0:
```
Comment (kaylynn234) [inline crimsobot/cogs/utilities.py:42]: Unsure why this snippet got past. This can be shifted to use `not` (`if not question`) for cleanliness.

File: crimsobot/cogs/utilities.py [hunk 2]
```diff
@@ -36,11 +36,16 @@ async def poll(self, ctx: commands.Context, *, poll_input: str) -> None:
...
             question = 'QUICK POLL!'
 
+        # message attachment if it exists
+        if len(ctx.message.attachments) > 0:
```
Comment (kaylynn234) [inline crimsobot/cogs/utilities.py:46]: Same as above; We can use `not` here.

File: crimsobot/cogs/utilities.py [hunk 3]
```diff
@@ -95,6 +103,10 @@ async def polltally(self, ctx: commands.Context, poll_id: Optional[str] = None)
                     poll_found = True
                     descr = message.embeds[0].description
                     reactions = message.reactions
+
+                    if message.embeds[0].image is not discord.Embed.Empty:
```
Comment (kaylynn234) [inline crimsobot/cogs/utilities.py:107]: If memory serves, Embed.Empty is a false-y value. `if not message.embeds[0].image` should be valid.

File: crimsobot/cogs/utilities.py [hunk 4]
```diff
@@ -95,6 +103,10 @@ async def polltally(self, ctx: commands.Context, poll_id: Optional[str] = None)
...
+                        imageurl = message.embeds[0].image.url  # this looks kind of yucky
```
Comment (kaylynn234) [inline crimsobot/cogs/utilities.py:108]: I don't like concatenating words with variable names and would rather separate them with an underscore per convention; i.e `image_url`, but this is up to choice.

**Review by kaylynn234 — COMMENTED** [review]
Just some non-blocking nits to do with code style. I don't see any real issues in this PR but another set of eyes could be good.

File: crimsobot/cogs/utilities.py (hunk 1 above)
Comment (keli5) [reply crimsobot/cogs/utilities.py:42]: Not sure why I put that down either. Removed.

**Review by williammck — APPROVED** [review]
